scripts/dtsBundler.mjs

```diff
@@ -409,5 +410,22 @@ if (publicContents.includes("@internal")) {
     console.error("Output includes untrimmed @internal nodes!");
 }
 
-fs.writeFileSync(output, publicContents);
-fs.writeFileSync(internalOutput, internalContents);
+/**
+ * @param {string} contents
+ * @returns {string}
+ */
+function dprint(contents) {
+    return cp.execFileSync(
+        path.resolve(__dirname, "..", "node_modules", "dprint", "bin.js"),
+        ["fmt", "--stdin", "ts"],

```

Comment:
This fixes the `ETXTBSY` error on Linux and makes it work on Windows:

```suggestion
        process.execPath,
        // needs `import { createRequire } from "module";` above
        [createRequire(import.meta.url).resolve("dprint/bin.js"), "fmt", "--stdin", "ts"],
```

Reply:
Whoops, that wasn't intentional!

Reply:
I had meant to run with execPath; the error in CI is now back to that wasm error it seems.

Reply:
Ah, I think I need some additional synchronization around multiple processes updating the wasm cache. Will fix it now.

Reply:
I fixed it in 0.38.2.

Reply:
Now the npm package initialization at the same time by multiple processes is fixed in 0.38.3 (uses a post install step and when scripts are disabled it will fall back to a lazy-initialization step that should work if multiple processes do it at the same time).

Reply:
I think it's all working now. Thanks!